celext/cel_test.go

```diff
@@ -119,10 +119,10 @@ func TestCelJSON(t *testing.T) {
 		Input  string
 		Output any
 	}{
-		{Input: `dyn([{'name': 'John', 'age': 30}]).toJSONArray()`, Output: `[{"age":30,"name":"John"}]`},
-		{Input: `[{'name': 'John'}].toJSONArray()`, Output: `[{"name":"John"}]`},
-		{Input: `dyn({'name': 'John'}).toJSON()`, Output: `{"name":"John"}`},
-		{Input: `{'name': 'John'}.toJSON()`, Output: `{"name":"John"}`},
+		{Input: `dyn([{'name': 'John', 'age': 30}]).marshalJSON()`, Output: `[{"age":30,"name":"John"}]`},
```

Comment:
Why not just toJSON ?

Reply:
just to replicate the go terminology, should I change it to `toJSON` ?

Reply:
yeh, because we have a few other functions toYAML|TOML|CSV etc..

Reply:
cool, will update